pages/signin.html

```diff
@@ -5,11 +5,53 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css">
+    <link rel="stylesheet" href="../style/style.css">
+    <title>Sign In</title>
 </head>
 
 <body>
+    <header>
+        <a class="logo" href="#">my<span>Bank</span></a>
 
+    </header>
+    <main>
+        <div class="mainSvg">
+            <h1> We Keep Customerers <span>Coming Back For More !</span> </h1>
+            <img src="../library/mainPageimage.svg" alt="" srcset="">
+        </div>
+        <div class="signInAria">
+            
+            <form onsubmit="return false;">
+                <div id="logo"><img src="../library/logo.svg" alt="logo"></div>
+
+                <div class="inputDiv">
+                    <label>Email</label>
+                    <i class="bi bi-person-fill"></i>
+                    <input type="email" placeholder="Email">
+                </div>
+                <div class="inputDiv">
+                        <label>Password</label>
+                        <i class="bi bi-lock-fill"></i>
+                        <input type="password" placeholder="Password">               
+                </div>
+
+                <div class="btnDiv">
+                    <p class="special_txt">Forgot password</p>
```

Comment:
change it to a link

Reply:
hey Hussin ,Thanks for you're commit
though in the beginning  I thought this is  really not matter if we match with the right "id" p-element. I fixed it.